nltk/test/unit/lm/test_vocabulary.py

```diff
@@ -149,5 +149,6 @@ def test_len_is_constant(self):
         small_vocab_len_time = timeit("len(small_vocab)", globals=locals())
         large_vocab_len_time = timeit("len(large_vocab)", globals=locals())
 
-        # The timing should be the same order of magnitude.
-        self.assertAlmostEqual(small_vocab_len_time, large_vocab_len_time, places=1)
+        # The timing should be the same order of magnitude. The delta is pretty
```

Comment:
I'm kinda against this. We're changing an important assertion about the software based only on convenience testing it...

Here's an idea for how to deal with this: `pytest` has this concept of markers, we could introduce one for tests that we don't want to run in parallel. In the CI we execute our parallel tests first, then follow up by executing the non-parallel ones. I'd definitely have a look that save us any time as a result, perhaps the additional complication isn't really worth it?

I'm guessing that it should be though, given that tests take quite a while currently.